Classes/Classes/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.10)
+
+set (CMAKE_BUILD_TYPE Debug)
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set (CMAKE_CXX_COMPILER "D:\\mingw\\bin\\mingw32-g++.exe")
```

Comment:
This is depending on the path where you have the compiler, do we actually need this? the best would be to rely on having this path added into the environment